README.md

```diff
@@ -131,6 +131,42 @@ type config struct {
 }
 ```
 
+
+## From file
+
+The `env` tag option `file` (e.g., `env:"tagKey,file"`) can be added
+to load the value from a file. The environment variable suffixed with `_FILE` is then expected and looked at for the path 
+to that file. If the suffixed version of the environment variable is not present, the value of the original environment 
```

Comment:
Thinking now, this auto `_FILE` thing is a bit confusing.

I propose:

- use the env name as the user gives it (in the example bellow, it would be SECRET_KEY)
- error if we cant load for any reason
- envDefault when the env is a file would be expected to be a file path as well

wdyt?

Reply:
I see you point that the behavior is not always clear. 
My intention was for this to be an overloading functionality as well and figured that convention over configuration might be easier to work with, or at least for my use case. 

The reasoning behind this is that I'm at the moment building a service where users deploying it them self should decide on if they want to load, in my case PEM files, from disk or from the environment. While letting the env lib load the files would help but there would then have to some code on my part figuring out what the correct variable to use is, the file or en env one.

I got the idea from https://docs.traefik.io/v1.7/configuration/acme/#provider where they let users load a whole lot of secrets from files or from env.

I, personally, want the overloading capability in order to provide two ways to configure one setting. But it could wok in other ways.

eg.
```go
SecretKey string `env:"SECRET_KEY,file"`  //loads file from path stored in env
// ...
SecretKey string `env:"SECRET_KEY,file" envSuffix="_FILE"`  //loads value from env, or value from file path in SECRET_KEY_FILE
```
I however think this behavior might make it even more confusing...

Thoughts?

Reply:
I still prefer not having this kind of unclear behavior.

the user can always add a `func (e MyStruct) SecretKey() string {}` to deal with this, or maybe even hide this detail in an inner struct and handle it when loading the variables...

Reply:
Hm.. i see what your getting at. Sure makes it some what more verbose. I'm guessing your getting at something like the following to accomplish the same thing?

```go
type Config struct {
	SecretKey SecretKey
}
type SecretKey struct {
	Env  string `env:"SECRET_KEY"`
	File string `env:"SECRET_KEY_FILE,file"`
}
func (s SecretKey) Value() string {
	if len(s.File) > 0{
		return s.File
	}
	return s.Env
}

func main() {
	cfg := Config{}
	if err := env.Parse(&cfg); err != nil {
		fmt.Printf("%+v\n", err)
	}
	fmt.Println(cfg.SecretKey.Value())
}

```

Reply:
yes, something like that...

as they say, clear is better than clever...

Reply:
updated to you liking